src/runtime/containerd-shim-v2/service.go

```diff
@@ -290,7 +303,7 @@ func getTopic(e interface{}) string {
 
 func trace(ctx context.Context, name string) (otelTrace.Span, context.Context) {
 	if ctx == nil {
-		logrus.WithField("type", "bug").Error("trace called before context set")
+		logrus.WithField("type", "bug").Errorf("called before context set: %s", name)
```

Comment:
This might be better as:

```suggestion
		logrus.WithFields(logrus.Fields{"type": "bug", "name": name}).Error("called before context set")
```